xbmc/guilib/guiinfo/GUIInfoLabels.h

```diff
@@ -173,27 +173,28 @@
 #define MUSICPLAYER_RATING          219
 #define MUSICPLAYER_COMMENT         220
 #define MUSICPLAYER_LYRICS          221
-#define MUSICPLAYER_HASPREVIOUS     222
-#define MUSICPLAYER_HASNEXT         223
-#define MUSICPLAYER_EXISTS          224
-#define MUSICPLAYER_PLAYLISTPLAYING 225
-#define MUSICPLAYER_ALBUM_ARTIST    226
-#define MUSICPLAYER_PLAYCOUNT       227
-#define MUSICPLAYER_LASTPLAYED      228
-#define MUSICPLAYER_CHANNEL_NAME    229
-// unused 230
-#define MUSICPLAYER_CHANNEL_GROUP   231
-// unused 232
+#define MUSICPLAYER_ALBUM_ARTIST    222
+#define MUSICPLAYER_PLAYCOUNT       223
+#define MUSICPLAYER_LASTPLAYED      224
+#define MUSICPLAYER_USER_RATING     225
+#define MUSICPLAYER_RATING_AND_VOTES     226
+#define MUSICPLAYER_VOTES           227
+#define MUSICPLAYER_MOOD            228
+#define MUSICPLAYER_CONTRIBUTORS    229
+#define MUSICPLAYER_CONTRIBUTOR_AND_ROLE 230
+#define MUSICPLAYER_CHANNEL_NAME    231
+#define MUSICPLAYER_CHANNEL_GROUP   232
 #define MUSICPLAYER_CHANNEL_NUMBER  233
```

Comment:
the three CHANNELS ones can be moved down (below MUSICPLAYER_PROPERTY)
as they don't support offset/position.

Reply:
Would it make sense to add support for offset/position for these three ?

Reply:
To answer myself: I don't think so, as they are only used for PVR radio channels, which are not organized in playlists.